.circleci/config.yml

```diff
@@ -132,6 +135,19 @@ create_conda_env: &create_conda_env
         python --version
         pip install --upgrade pip
 
+download_and_configure_125m_with_hf_dependencies: &download_and_configure_125m_with_hf_dependencies
+  - run:
+      name: Download and configure a 125m checkpoint with HF dependencies
+      working_directory: ~/metaseq/gpu_tests
+      command: |
+        source activate metaseq
+        if [[ $(python -c "import torch; print(torch.__version__)") = "1.9.1+cu111" ]]; then
```

Comment:
The metaseq model loading doesn't seem to work well with pytorch 1.8, so only running the test for pt19

Reply:
We can kill pytorch 1.8. It's behind and internally we're using 1.10

Reply:
Let's create an issue for that for the future.